packages/engine/src/renderer/EngineRendererState.ts

```diff
@@ -62,53 +63,59 @@ function setUsePostProcessing(usePostProcessing) {
 export class EngineRendererReceptor {
   static setQualityLevel(action: typeof EngineRendererAction.setQualityLevel.matches._TYPE) {
     const s = getState(EngineRendererState)
-    s.merge({ qualityLevel: action.qualityLevel })
+    s.qualityLevel.set(action.qualityLevel)
```

Comment:
I assume this is an unrelated fix?

Reply:
yes, fixed the debuggers to make it easier to debug stuff